incubator/mysqlha/templates/statefulset.yaml

```diff
@@ -14,6 +14,10 @@ spec:
     metadata:
       labels:
         app: {{ template "fullname" . }}
+      {{- if .Values.mysqlha.podAnnotations }}
```

Comment:
I haven't done YAML templating in awhile, but with the `podAnnotations` defined in the values.yaml, does this make the statement always true?

Reply:
By default `podAnnotations` are set to empty mapping in values.yaml, so it will not be true.